### PR TITLE
Changed the implementation of new Sales Area page to modal form in bo…

### PR DESCRIPTION
**Pull request by Priyanka-Kaintura**
Targets #2356 

### Description
Changed the implementation of new Sales Area page to bootstrap modal forms.

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/SalesAutomation/Resources/views/sales-area/index.blade.php [hunk 1]
```diff
@@ -52,8 +52,51 @@
...
-	    </thead>
+				@empty
+				<tr>
+                    <td colspan="2">
+						<div class="d-flex justify-content-center">
+							<img src="{{ URL('images/no-result.png') }}" class='img-center' alt="" width="50%">
+						</div>
+						<div class="container-fluid d-flex justify-content-center">
+							<span class="text-primary">NO DATA FOUND</span>
+						</div>
```
Comment (Ayush8923) [inline Modules/SalesAutomation/Resources/views/sales-area/index.blade.php:63]: Please fix the indentation.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
@Priyanka-Kaintura Left some comments. 
Also please add the toast when the user creates the new sales area.

**Comment by Ayush8923** [comment]
@Priyanka-Kaintura Please take the latest pull of the `master` branch to this branch. So that both the branches go in sync up with each other

File: Modules/SalesAutomation/Resources/views/sales-area/index.blade.php [hunk 2]
```diff
@@ -52,8 +53,51 @@
...
+	<div class="modal-dialog">
+		<div class="modal-content">
+			<div class="modal-title d-flex justify-content-between mx-2 mb-0 mt-2">
+				<h2>New Sales Area</h2>
+				<button class="btn btn-secondary" data-dismiss="modal"><span>&times</span></button>
+			</div>
+			<form action="{{ route('sales-area.store') }}" method="POST">
+			@csrf
+				<div class="modal-body mt-1">
+					<div class="card my-0">
```
Comment (Ayush8923) [inline Modules/SalesAutomation/Resources/views/sales-area/index.blade.php:83]: We don't need to make a form as a card. which makes it look a bit weird then.
Take the reference of the below screenshot. The shadow of the form is visible in the background of the form. we need to see the normal form. also please take a reference CodeTrek applicant form.
image

File: Modules/SalesAutomation/Resources/views/sales-area/index.blade.php [hunk 3]
```diff
@@ -52,8 +53,51 @@
...
 	{{ $salesAreas->links() }}
-@endsection
\ No newline at end of file
+@endsection
+
+<div class="modal fade" id="newSalesArea">
+	<div class="modal-dialog">
+		<div class="modal-content">
+			<div class="modal-title d-flex justify-content-between mx-2 mb-0 mt-2">
+				<h2>New Sales Area</h2>
```
Comment (Ayush8923) [inline Modules/SalesAutomation/Resources/views/sales-area/index.blade.php:77]: This text size is too large in the model form. Please take the reference other places.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Added a few comments. Please check.

File: Modules/SalesAutomation/Resources/views/sales-area/index.blade.php (hunk 3 above)
Comment (Priyanka-Kaintura) [reply Modules/SalesAutomation/Resources/views/sales-area/index.blade.php]: @Ayush8923 Sir, could you please check it now.

File: Modules/SalesAutomation/Resources/views/sales-area/index.blade.php [hunk 4]
```diff
@@ -52,8 +53,43 @@
 						</form>
 					</td>
 				</tr>
-			@endforeach		
-	    </thead>
+				@empty
+				<tr>
+                    			<td colspan="2">
```
Comment (LavishRawat) [inline Modules/SalesAutomation/Resources/views/sales-area/index.blade.php:58]: @Priyanka-Kaintura i can not see the closing tag for `` <td ``. Please have a look.

**Review by LavishRawat — CHANGES_REQUESTED** [review]

**Comment by LavishRawat** [comment]
@Priyanka-Kaintura  in the UI can see a white box at the end of the table. Please have a look.

Screenshot 2023-06-23 at 10 40 09 AM

**Review by LavishRawat — APPROVED** [review]

File: Modules/SalesAutomation/Resources/views/sales-area/index.blade.php [hunk 5]
```diff
@@ -52,8 +53,43 @@
...
+						<div class="d-flex justify-content-center">
+							<img src="{{ URL('images/no-result.png') }}" class='img-center' alt="" width="50%">
```
Comment (Ayush8923) [inline Modules/SalesAutomation/Resources/views/sales-area/index.blade.php:60]: Remove the unwanted attributes.
```suggestion
							<img src="{{ URL('images/no-result.png') }}" class='img-center' width="50%">
```

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
One small comment else looks good.

**Review by Ayush8923 — APPROVED** [review]

**Pull request merged**